fdbbackup/backup.actor.cpp

```diff
@@ -375,6 +377,7 @@ CSimpleOpt::SOption g_rgBackupAbortOptions[] = {
 	{ OPT_HELP, "--help", SO_NONE },
 	{ OPT_DEVHELP, "--dev-help", SO_NONE },
 	{ OPT_KNOB, "--knob-", SO_REQ_SEP },
+	{ OPT_SWIFT_TEST_FILTER, "--swift-test-filter", SO_NONE },
```

Comment:
Same as above. If used, many other option list in this file should also be added, I think.